RELEASE/scripts/autoscend.ash

```diff
@@ -4449,77 +4449,97 @@ boolean LX_bitchinMeatcar()
 		set_property("lastDesertUnlock", my_ascensions());
 		return false;
 	}
-
-	int meatRequired = 100;
-	if(item_amount($item[Meat Stack]) > 0)
-	{
-		meatRequired = 0;
-	}
-	foreach it in $items[Spring, Sprocket, Cog, Empty Meat Tank, Tires, Sweet Rims]
+	
+	//craft meatcar if you have all the components. note that you also need 70 meat for 7 meat pastes if not in knoll sign
+	if(!knoll_available() && item_amount($item[Tires]) > 0 && item_amount($item[spring]) > 0 && item_amount($item[sprocket]) > 0 && item_amount($item[cog]) > 0 && item_amount($item[empty meat tank]) > 0)
```

Comment:
I think this is missing the condition for sweet rims

Reply:
You are right. oops.
obsoleted by commit fd192a2 which uses cleaner code to handle acquisition of car